supplementary/alica_codegeneration/README.md

```diff
@@ -0,0 +1,29 @@
+# ALICA Codegeneration
+
+This is the ALICA Codegeneration package.
+
+## Build
+
+To build the ALICA codegeneration jar files, run `mvn -DskipTests=true install` inside the root folder of alica_codegeneration.
+
+After building the jar files using maven, move the codegeneration jar file `alica_codegeneration/target/PlanDesignerFX-Codegeneration-0.1.1.X.jar` to `/alica/supplementary/alica_designer_runtime/codegen/`. Next, move the default plugin `alica_codegeneration/alica-plan-designer-fx-default-plugin/target/alica-plan-designer-fx-default-plugin-0.1.1-SNAPSHOT.jar` to `/alica/supplementary/alica_designer_runtime/codegen/plugins/`.
+
```

Comment:
@david-simoes-93 I guess this covers a lot of what we need to do to make this automatic